changelogs/README.md

```diff
@@ -0,0 +1,14 @@
+## Adding a changelog
+
+Pull request authors are expected to include a changelog file explaining the changes introduced by the pull request. 
+The changelog file should be a new file created in the `changelogs/unreleased` folder. 
```

Comment:
What's the advantage of having changelogs in different files which are then combined rather than modifying the changelog directly?

Reply:
It could cause conflicts a lot, here's a good explanation https://docs.gitlab.com/ee/development/changelog.html#history-and-reasoning

Reply:
I see. Some things I'm wondering then:
- how do people know the format of these individual changelog files? I think your source has a cli that takes care of that, seems a bit overkill for us?
- Would we realistically run into a lot of the conflicts this aims to avoid? And even if they happen, are they hard to fix?
- Isn't it possible to just check that changelog file changed? Or are you talking about format checking?

Reply:
In our case, a changelog entry is just a raw text explaining the changes (no specific format). Gitlab uses a CLI tool to generate YAML files and it needs a tool to parse them later to generate an actual changelog, which I wanted to avoid as it is an overkill, I agree). I think conflicts should be easy to fix, but we should have a way to avoid them completely, as they might add unnecessary delays.

Another problem is if we modify a changelog directly, we cannot make sure it follows the same format (we can check if a file changed in CI using git diff, but then the actual format must be checked manually by us on each PR that modified a changelog)

Reply:
Alright, let's give this a go then!